src/formatter/Cli.hx

```diff
@@ -204,8 +204,9 @@ class Cli {
 			if (verbose) {
 				verboseLogFile(path, config);
 			}
+			var filePath = new Path(path);
 			var content:String = File.getContent(path);
-			var result:Result = Formatter.format(Code(content), config);
+			var result:Result = Formatter.format(Code(content, SourceFile(filePath.file)), config);
```

Comment:
I think we should pass full `path` instead of just filename in case users have identical filenames in different subfolders

Reply:
Good point I'll change this.